assets/my_bookings_show.js

```diff
@@ -5,7 +5,7 @@ let formData = new FormData();
 let bookingId;
 let newBookingStartAt;
 let newBookingEndAt;
-moment.locale('fr');
+let user = document.getElementById('user').dataset.userId;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/LeonardDeVanGogh/VosgesActivies/pullRequest?prid=7268173)
```suggestion
let user = document.getElementById("user").dataset.userId;
```